custom-checks/checkstyle/src/test/java/org/openhab/tools/analysis/checkstyle/test/MarkdownCheckTest.java

```diff
@@ -66,7 +69,7 @@ public void testSpecialHeader() throws Exception {
 
     @Test
     public void testEmptyLineBeforeList() throws Exception {
-        String[] expectedMessages = generateExpectedMessages(1, "The line before a Markdown list must be empty.");
+        String[] expectedMessages = generateExpectedMessages(2, "The line before a Markdown list must be empty.");
```

Comment:
Why do only some of the linenumbers change is that a bug or a feature?

Reply:
It is not a bug. After changing the parsing library and adding more tests, the logging in some cases seemed inconsistent, so we changed where the warning message should be placed.